docs/source/simulated_populations/index.rst

```diff
@@ -67,10 +67,10 @@ The following table provides the SHA-256 checksum for the larger-scale simulated
     - SHA-256 checksum
   * - US
     - pseudopeople_simulated_population_usa_1_0_1.zip
-    - 9462cc60b333fb2a3d16554a9e59b5428a81a2b1d2c34ed383883d7b68d2f89f
+    - 0025978196c2a84c1df502e857bec35a84c25092fbfb6b143c0b8ff30dea5eed
```

Comment:
Yesterday we discussed two relevant things at the RT/SE:
1. Keeping a table of all previous checksums rather than just replacing them with the current version
2. Changing the version numbers of both the data and pseudopeople to be 2.0.0 in order to keep them in better sync

Do we want to proceed with those two proposals? If so, we should make the appropriate changes to the checksum table here.

Reply:
Agree with (1), and re: (2) can you please check that the file _name_ is identical overall to what was uploaded to Azure?

Reply:
I can keep the old versions in this table and add 2.0.0